SpatialGDK/Source/SpatialGDK/Private/Interop/ActorSystem.cpp

```diff
@@ -199,15 +199,33 @@ void ActorSystem::ProcessAdds(const FEntitySubViewUpdate& SubViewUpdate)
 			{
 				RefreshEntity(Delta.EntityId);
 			}
+		}
+	}
+}
 
-			if (SubViewUpdate.SubViewType != ENetRole::ROLE_SimulatedProxy)
+void ActorSystem::ProcessAuthorityGains(const FEntitySubViewUpdate& SubViewUpdate)
```

Comment:
Do you need to pass the subview in? Presumably this is only valid with the auth subview

Reply:
I think we also need to do this for the OwnershipSubView, no?

Reply:
but good point, below I'm iterating over all three subviews, so can remove one of those to save time